Objective-C/Tests/Util/CBLWordEmbeddingModel.m

```diff
@@ -40,7 +40,8 @@ - (CBLDictionary*) predict: (CBLDictionary*)input {
     NSString* inputWord = [input stringForKey: @"word"];
     
     if (!inputWord) {
-        NSLog(@"No word input !!!");
+        os_log_t log = os_log_create("CouchbaseLite", "OSLogging");
+        os_log(log, "No word input !!!");
```

Comment:
For this one we can use NSLog, I guessed or change it to assertion instead of logging.

Reply:
will change to assert